.github/workflows/auto-publish.yml

```diff
@@ -51,20 +55,24 @@ jobs:
        - name: Export to packwiz
          run: |
            chmod +x ./packwiz
-           pushd Packwiz/${{ vars.MC_VERSION }}
-           ../../packwiz modrinth export -o ../../Fabulously.Optimized-${{ github.ref_name }}.mrpack
+           pushd Packwiz/${VARS_MC_VERSION}
+           ../../packwiz modrinth export -o ../../Fabulously.Optimized-${GITHUB_REF_NAME}.mrpack
            popd
+         env:
+           VARS_MC_VERSION: ${{ vars.MC_VERSION }}
```

Comment:
What kind of MITM attacks would this section prevent, considering it already is and still will be a variable?